frontend/src/metabase-lib/expressions/helper-text-strings.ts

```diff
@@ -822,6 +822,7 @@ const HELPER_TEXT_STRINGS: HelpTextConfig[] = [
       },
       {
         name: t`mode`,
+        // TODO: This is the only place that's not easy to replace the application name.
```

Comment:
![Screenshot 2023-12-07 at 6 06 05 PM](https://github.com/metabase/metabase/assets/1937582/dc2bf8a2-e448-4bf0-be7d-74d8d4fbedf2)

Reply:
Considering the edge case (not being in a component), should we use the deprecated ` MetabaseSettings.get("application-name")`?
It's deprecated, but it should work fine for now

In alternative we could export the store and do `getSetting(store.getState,'application-name')` but it doesn't seem we currently export the store so it's maybe worth doing in another PR

Reply:
I don't want to introduce anything that uses `MetabaseSettings` since we're migrating away from it. I know the migration won't be completed any time soon. But by keep using it, it really doesn't help.

Also consider that this is only a single place I found so far I think it's worth leaving this as it probably won't affect a lot of users at scale. And for simplicity in our code, I don't think it's worth trying to juggle things around just to make this one place have a proper application name replacement.

Maybe @albertoperdomo could chime in and give product's perspective. But from engineering's point of view, it's not worth the effort trying to fix this one place.

What do you guys think?

Reply:
If we ever decided to fix this. I can introduce it in another PR since I'd rather have this one merged now rather than later to avoid merge conflicts.